### PR TITLE
feat: add vite-plugin-swc-only

### PR DESCRIPTION
**Pull request by natrim**
> Please make sure all the requirements are satisfied, otherwise the PR could be closed without further notice.

## Checklist

- [x] Title as described.
- [x] Make sure you put things in the right category.
- [x] The description of your item should be a sentence with less than 24 words.
- [x] Avoid using links and parentheses in description. 
- [x] Omit unnecessary words already provided in the context (e.g. `for Vite`, `a Vite plugin`).
- [x] Use proper case for terms (e.g. `GitHub`, `TypeScript`, `ESLint`, etc.)
- [x] When mentioning tools, omit versions whenever possible (e.g. use `TypeScript` instead of `TypeScript 4.x`, but keep `Vue 2` and `Vue 3` as they're incompatible).
- [x] When mentioning package names, use quotes whenever possible.
- [x] Always add your items to the end of a list.

### Plugins/Tools

- [x] The plugin/tool is working with **Vite 2.x and onward**.
- [x] The project is Open Source.
- [x] The project follows the [Vite Plugins Conventions](https://vitejs.dev/guide/api-plugin.html#conventions).
- [x] The plugin uses Vite-specific hooks and can't be implemented as a [Compatible Rollup Plugin](https://vitejs.dev/guide/api-plugin.html#rollup-plugin-compatibility).
- [x] The repo should be at least 30 days old.
- [x] The documentation is in English.
- [x] The project is active and maintained (inactive projects for longer 6 months will be removed without further notice).
- [x] The project accepts contributions.
- [x] Not a commercial product.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -406,6 +405,12 @@ In this section, we use badges to indicate the targeted Vue version for each plu
 
 #### Transformers
 
+- [vite-plugin-swc-only](https://github.com/natrim/vite-plugin-swc-only) - Use SWC to transform and minify app.
+
+#### Helpers
+
+- [vite-plugin-react-refresh](https://github.com/fengxinming/vite-plugins/tree/main/packages/vite-plugin-react-refresh) - Provide enhancements for [@vitejs/plugin-react-refresh](https://www.npmjs.com/package/@vitejs/plugin-react-refresh).
```
Comment (Scrum) [inline README.md:412]: I don’t think it makes sense to single out a separate category, perhaps a subcategory is better

**Review by Scrum — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (natrim) [reply README.md:412]: ?? it's not new category, its the same subcategory as Vue has above

Comment (Scrum) [reply README.md:412]: I'm talking about analogue https://github.com/vitejs/awesome-vite#react-1

**Review by Scrum — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -406,6 +405,9 @@ In this section, we use badges to indicate the targeted Vue version for each plu
 
 #### Transformers
 
+- [vite-plugin-swc-only](https://github.com/natrim/vite-plugin-swc-only) - Use SWC to transform and minify app.
+- [vite-plugin-react-refresh](https://github.com/fengxinming/vite-plugins/tree/main/packages/vite-plugin-react-refresh) - Provide enhancements for [@vitejs/plugin-react-refresh](https://www.npmjs.com/package/@vitejs/plugin-react-refresh).
```
Comment (Scrum) [inline README.md:409]: why did you move it?

**Review by Scrum — CHANGES_REQUESTED** [review]
